solidus_page_objects.gemspec

```diff
@@ -31,4 +31,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'rubocop-rspec', '1.13.0'
   s.add_development_dependency 'simplecov'
   s.add_development_dependency 'sqlite3'
+  s.add_development_dependency 'pry-byebug'
```

Comment:
because I think this is an optional gem and there could be someone that doesn't need it but will be forced to have it installed without a real reason. IMO this is not a dependency but a personal preference. If you prefer to leave it I can leave with it, it was just my opinion 😄